go-controller/pkg/kubevirt/switch.go

```diff
@@ -0,0 +1,22 @@
+package kubevirt
+
+import (
+	"net"
+	"strings"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+const (
+	ARPProxyIPv4 = "169.254.1.1"
+	ARPProxyIPv6 = "d7b:6b4d:7b25:d22f::1"
+)
+
+// ComposeARPProxyLSPOption returns the "arp_proxy" field needed at router type
+// LSP to implement stable default gw for pod ip migration, it consiste of
+// generated MAC address and a link local ipv4 and ipv6, it's the same
+// for all the logical switches.
+func ComposeARPProxyLSPOption() string {
+	mac := util.IPAddrToHWAddr(net.ParseIP(ARPProxyIPv4)).String()
+	return strings.Join([]string{mac, ARPProxyIPv4, ARPProxyIPv6}, " ")
+}
```

Comment:
this is good, but maybe if we exported a type it would become more readable.

What do you think of:

```golang
type ARPProxy struct {
    mac string
}

func NewARPProxy() *ARPProxy {
    return &ARPProxy{mac: util.IPAddrToHWAddr(net.ParseIP(ARPProxyIPv4)).String()}
}

func (ap *ARPProxy) LSPOption string {
    return strings.Join([]string{mac, ARPProxyIPv4, ARPProxyIPv6}, " ")
}
```

Feel free to ignore, this might be a bit too much.